### PR TITLE
Release/one more time

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of Leather — [download extension builds](https://github.com/leather-wallet/extension/actions/runs/6381478842).

Another release

### CODE REVIEW

File: .github/workflows/gh-pages.yml [hunk 1]
```diff
@@ -16,9 +16,6 @@ jobs:
       - name: Build prod
         run: yarn build
 
-      - name: Run jest
```
Comment (pete-watters) [inline .github/workflows/gh-pages.yml:19]: Do we want to never run the unit tests in CI now? (the vitest ones)

Comment (kyranjamie) [reply .github/workflows/gh-pages.yml:19]: Tests are ran here, but this indeed breaks the hosted code coverage site https://github.com/leather-wallet/extension/blob/b56c27ba5e5f8da85a9582340671aef0ba1c584f/.github/workflows/code-checks.yml#L113-L123

will make an issue to fix, but it's non-critical

File: tsconfig.json [hunk 2]
```diff
@@ -29,21 +29,14 @@
       "@inpage/*": ["inpage/*"],
       "@app/*": ["app/*"],
       "@tests/*": ["../tests/*"],
-      "@tests-legacy/*": ["../tests-legacy/*"],
+
```
Comment (pete-watters) [inline tsconfig.json:32]: Space 😱

**Review by pete-watters — APPROVED** [review]

File: tsconfig.json (hunk 2 above)
Comment (kyranjamie) [reply tsconfig.json:32]: will fix separately, only rushing bc firefox 🤦🏼

**Pull request merged**